packages/appeals-service-api/src/lib/get-address.js

```diff
@@ -0,0 +1,14 @@
+const getAddress = (siteAddress) =>
+  [
+    siteAddress?.addressLine1,
+    siteAddress?.addressLine2,
+    siteAddress?.town,
+    siteAddress?.county,
+    siteAddress?.postcode,
+  ].filter((n) => n);
+
+module.exports = {
+  getAddress,
+  getAddressSingleLine: (siteAddress) => getAddress(siteAddress).join(', '),
```

Comment:
I didn't know you could declare functions in export like this. Nice for one-liners.